package.json

```diff
@@ -52,6 +53,7 @@
     "eslint-plugin-standard": "^4.0.1",
     "jest": "^24.9.0",
     "jsdoc-to-markdown": "^6.0.1",
-    "memfs": "^3.0.1"
+    "memfs": "^3.0.1",
+    "superagent": "^6.1.0"
```

Comment:
this should be a dev dependency

Reply:
@moritzraho It is part of devDependencies